src/main/java/org/sonar/squidbridge/rules/ExternalDescriptionLoader.java

```diff
@@ -23,12 +23,16 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Charsets;
 import com.google.common.io.Resources;
-import org.sonar.api.server.rule.RulesDefinition.NewRepository;
-import org.sonar.api.server.rule.RulesDefinition.NewRule;
-
 import java.io.IOException;
 import java.net.URL;
+import org.sonar.api.server.rule.RulesDefinition.NewRepository;
+import org.sonar.api.server.rule.RulesDefinition.NewRule;
 
+/**
+ * @deprecated since 2.7. will be removed without alternatives
+ * Responsibility of loading rule description is let to plugins
+ */
+@Deprecated
 @Beta
 public class ExternalDescriptionLoader {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlqEaf12pdu-6jFC&open=AWHXxlqEaf12pdu-6jFC)

[ISSUE_KEY]: AWHXxlqEaf12pdu-6jFC